.github/workflows/dry-run.yaml

```
@@ -0,0 +1,33 @@
+name: OctoDNS Dry Run
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+
+      - name: Install dependencies
+        run: |
+          python -m venv venv
+          source venv/bin/activate
+          pip install -r requirements.txt
+
+      - name: Run OctoDNS dry run
+        run: |
+          source venv/bin/activate
+          octodns-sync --config-file=./config/production/octodns.yaml
```

```suggestion
          octodns-sync --config-file=./config/octodns.yaml
```
No production directory

---

Absolutely. It was a sloppy copy and paste error.